images/virtualization-artifact/pkg/controller/cvi/cvi_webhook.go

```diff
@@ -68,7 +79,11 @@ func (v *Validator) ValidateUpdate(_ context.Context, oldObj, newObj runtime.Obj
 		return nil, fmt.Errorf("ClusterVirtualImage is in a Ready state: configuration changes are not available")
 	}
 
-	return nil, nil
+	if strings.Contains(newCVI.ObjectMeta.Name, ".") {
+		warnings = append(warnings, "cluster virtual image name contain '.', it may be cause of problems in future, please recreate resource.")
```

Comment:
"Recreate" is a creation with the same name. I think it's not what you meant =)

```suggestion
		warnings = append(warnings, "ClusterVirtualImage name is invalid as it contains now forbidden symbol '.', allowed symbols for name are [0-9a-zA-Z-]. Create another image with valid name to avoid problems with future updates.")
```